discord/abc.py

```diff
@@ -169,9 +169,9 @@ class _Overwrites:
 
     def __init__(self, **kwargs):
         self.id = kwargs.pop('id')
-        self.allow = int(kwargs.pop('allow_new', 0))
-        self.deny = int(kwargs.pop('deny_new', 0))
-        self.type = sys.intern(kwargs.pop('type'))
+        self.allow = int(kwargs.pop('allow', 0))
+        self.deny = int(kwargs.pop('deny', 0))
+        self.type = int(kwargs.pop('type'))
```

Comment:
They are necessary for allow and deny as they are string-serialized integers now. Also see the [permissions documentation](https://discord.com/developers/docs/topics/permissions) that makes an explicit remark about this. This is probably because they (will) go beyond the standard 32 bit integers of some programming languages.